bot/middlewares/any_middleware.py

```diff
@@ -23,14 +27,24 @@ def __init__(self, logger: logging.Logger, supported_commands: List[str]):
 
     async def __call__(
             self,
-            handler: Callable[[TelegramObject, Dict[str, Any]], Awaitable[Any]], event: TelegramObject,
+            handler: Callable[[TelegramObject, Dict[str, Any]], Awaitable[Any]],
+            event: TelegramObject,
             data: Dict[str, Any],
     ) -> Optional[Any]:
         if isinstance(event, Message):
             command = self.get_command_without_initial_slash(event)
+
             if command in self.__supported_commands:
                 self._logger.info(f"Command '{command}' accessed by user {event.from_user.id}")
 
+            if event.from_user:
+                user_id = event.from_user.id
+
+                is_admin_or_moderator = await DatabaseManager.is_admin_or_moderator(user_id)
+                if not is_admin_or_moderator and await DatabaseManager.is_command_limited(user_id, settings.MESSAGE_LIMIT, settings.LIMIT_DURATION):
+                    await event.answer(get_limit_exceeded_message())
+                    return None
+
```

Comment:
nie do konca, to musi byc w kazdym middleware, kazda komenda musi miec to sprawdzone.

to co sie tu dzieje powinno byc wyniesione jako funkcja klasy bazowej `BotMiddleware` i uzywane w kazdym middleware